dev/src/dev/render_png.clj

```diff
@@ -2,13 +2,18 @@
   "Improve feedback loop for dealing with png rendering code"
   (:require [clojure.java.io :as io]
             [clojure.java.shell :as sh]
+            [clojure.string :as str]
+            [hiccup.core :as h]
             [metabase.models.card :as card]
             [metabase.models.user :as user]
             [metabase.pulse :as pulse]
             [metabase.pulse.render :as pulse-render]
+            [metabase.pulse.render.js-svg :as poc]
```

Comment:
I feel like this is one of those theoretically-temporary acts of naming that will stay for an eyepopping amount of time if we don't think of another name besides `poc` right now. Maybe just "jssvg"? "js-render"?

Reply:
yeah i missed this rename. thanks